Cargo.toml

```diff
@@ -62,6 +62,7 @@ with-asan = ["libsqlite3-sys/with-asan"]
 column_decltype = []
 wasm32-wasi-vfs = ["libsqlite3-sys/wasm32-wasi-vfs"]
 winsqlite3 = ["libsqlite3-sys/winsqlite3"]
+os-other = ["libsqlite3-sys/os-other"]
```

Comment:
I think we want a better name for this. We possibly should even detect it from the build.rs based on the value of `std::env::var("TARGET")`...